app/code/Magento/InventoryInStorePickup/Model/Order/Email/ReadyForPickupSender.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickup\Model\Order\Email;
+
+use Magento\Framework\DataObject;
+
+/**
+ * Class ReadyForPickupSender
+ *
+ * @package Magento\InventoryInStorePickup\Model\Order\Email
+ * TODO: refactor
+ * TODO: Implement asynchronous email sending
```

Comment:
These two are important

Reply:
I will do it. But later. In a separate issue/PR. During asynchronous emails support implementation.